packages/landing/src/app/globals.css

```diff
@@ -70,6 +70,24 @@
   }
 }
 
+@font-face {
+  font-family: 'PlusJakartaSans';
+  font-display: swap;
+  src: url('/fonts/PlusJakartaSans-VariableFont_wght.ttf') format('truetype');
+}
+@font-face {
+  font-family: 'PlusJakartaSans';
+  font-display: swap;
+  font-weight: 700;
+  src: url('/fonts/PlusJakartaSans-Bold.ttf') format('truetype');
+}
+@font-face {
+  font-family: 'PlusJakartaSans';
+  font-display: swap;
+  font-weight: 500;
+  src: url('/fonts/PlusJakartaSans-Medium.ttf') format('truetype');
```

Comment:
Let's use `next/font/local` for better performance. https://nextjs.org/docs/pages/building-your-application/optimizing/fonts#local-fonts

Reply:
Ok, fixed in next commit

Reply:
Thank you